Content.Server/Atmos/TileAtmosphere.cs

```diff
@@ -0,0 +1,718 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+using Content.Server.GameObjects.EntitySystems;
+using Content.Server.Interfaces.Atmos;
+using Content.Shared.Atmos;
+using Content.Shared.GameObjects.EntitySystems;
+using NFluidsynth;
+using Robust.Shared.GameObjects;
+using Robust.Shared.GameObjects.Components;
+using Robust.Shared.GameObjects.Systems;
+using Robust.Shared.Interfaces.GameObjects;
+using Robust.Shared.Interfaces.Map;
+using Robust.Shared.Interfaces.Random;
+using Robust.Shared.IoC;
+using Robust.Shared.Map;
+using Robust.Shared.Maths;
+using Robust.Shared.Random;
+using Robust.Shared.Utility;
+using Logger = Robust.Shared.Log.Logger;
+using Math = CannyFastMath.Math;
+using MathF = CannyFastMath.MathF;
+
+namespace Content.Server.Atmos
+{
+    public class TileAtmosphere
+    {
+        private static long _eqQueueCycleCtr = 0;
```

Comment:
Not a fan of this static variable.

Reply:
Changed this to be a part of the grid atmos component.